src/lib/core/tweaks.h

```diff
@@ -0,0 +1,226 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2023, Tarantool AUTHORS, please see AUTHORS file.
```

Comment:
Just curious: `Copyright 2010-2023` here and `Copyright 2022-2023` in src/lua/tweaks.c. What is more correct or better in some sense?

Reply:
My bad. I blindly copied the copyright header from `src/lua/compat.h`, which used `2022-2022` for some reason. It looks like we use `2010-2023` everywhere else. Fixed.

<details><summary>diff</summary>

```diff
diff --git a/src/lua/tweaks.c b/src/lua/tweaks.c
index b8ae1ba38048..8a3c8045fcc8 100644
--- a/src/lua/tweaks.c
+++ b/src/lua/tweaks.c
@@ -1,7 +1,7 @@
 /*
  * SPDX-License-Identifier: BSD-2-Clause
  *
- * Copyright 2022-2023, Tarantool AUTHORS, please see AUTHORS file.
+ * Copyright 2010-2023, Tarantool AUTHORS, please see AUTHORS file.
  */
 #include "lua/tweaks.h"
 
diff --git a/src/lua/tweaks.h b/src/lua/tweaks.h
index 5ad429dde0fb..1e5455727b29 100644
--- a/src/lua/tweaks.h
+++ b/src/lua/tweaks.h
@@ -1,7 +1,7 @@
 /*
  * SPDX-License-Identifier: BSD-2-Clause
  *
- * Copyright 2022-2023, Tarantool AUTHORS, please see AUTHORS file.
+ * Copyright 2010-2023, Tarantool AUTHORS, please see AUTHORS file.
  */
 #pragma once
 
```

</details>